pkg/reading/service_read_issue.go

```diff
@@ -78,23 +77,31 @@ func (s *ServiceReadIssue) Name() string {
 	return "ServiceReadIssue"
 }
 
-func (s *ServiceReadIssue) ExecuteAsLastStep(ctx configuration.Context) error {
+func (s *ServiceReadIssue) ExecuteAsLastStep(params configuration.JiraAPIResourceParameters) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceReadIssue.ExecuteAsLastStep should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5050785)